frontend/src/main.ts

```diff
@@ -28,6 +27,14 @@ import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
 import { faTrashCan } from '@fortawesome/free-regular-svg-icons';
 import ErrorService from './services/ErrorService';
 
+import PrimeVue from 'primevue/config';
+
+// use bootstrap4 as default style
+import 'primevue/resources/themes/bootstrap4-light-blue/theme.css';
```

Comment:
Is bootstrap5 supported? Just wondering...

Reply:
Hi Ian, no primevue doesn't support bootstrap 5, it has a list of [built in theme](https://primevue.org/theming/#builtinthemes), but since most of us like bootstrap more, we just picked that one. Hopefully later it can provide bootstrap 5, but since we need customization anyway, should be good for now

Reply:
Sounds good, thanks for explanation!